content/docs/user-guide/editor/entity-outliner.md

```diff
@@ -15,5 +17,92 @@ The **Entity Outliner** shows all the entities and prefabs in the level. You can
 
    You can select multiple entities by doing one of the following in the **Entity Outliner**:
 
-   * Use **Ctrl + Left Mouse Button** to select multiple entities one at a time to be part of a multi-selection.
-   * Use **Shift + Left Mouse Button** to select a range of entities.
+- Use **Ctrl + Left Mouse Button** to select multiple entities one at a time to be part of a multi-selection.
+- Use **Shift + Left Mouse Button** to select a range of entities.
+
+## Reordering Entities
+
+When you create entities or instantiate prefabs, they appear at the bottom of the list in the **Entity Outliner**.
+
+1. To move an entity, right-click an entity and choose **Move up** or **Move down**.
+
+     ![Move an entity up and down the Entity Outliner.](/images/user-guide/component/entity_system/component-entity-outliner-reorder.png)
+
+1. You can also drag and drop one or more entities into the preferred order. Select and drag the entity until a white line appears in your preferred location.
+
+     ![Drag and drop to reorder an entity.](/images/user-guide/component/entity_system/component-entity-outliner-reorder-drag-drop.png)
+
+1. To make an entity the child of another entity, select and drag the entity to its intended parent. A white box appears around the parent entity.
+
+     ![Drag and drop to parent an entity.](/images/user-guide/component/entity_system/component-entity-outliner-parenting-drag-drop.png)
+
+## Hiding and Showing Entities
+
+You can hide entities so that they don't appear in the viewport, so that the viewport shows only the entities that you want. If you hide a parent entity, all children entities are also hidden. You can also hide child entities within a parent entity.
+
+1. In the **Entity Outliner**, click the eye icon next to the entity. The crossed-out eye icon indicates that the entity doesn't appear in the viewport.
+
+1. To show the entity, click the icon again.
+
+     ![Hide and show entities in the viewport.](/images/user-guide/component/entity_system/component-entity-outliner-hiding.png)
+
+## Locking Entities 
+
+You can lock entities so that they can't be selected in the viewport, so that you can select only the entities that you want to edit. If you lock a parent entity, all children entities are also locked. You can also lock child entities within a parent entity.
```

Comment:
suggestion: There's two 'so that...' here in quick succession. It might be nicer to say...

> 'You can lock entities so they can't be selected in the viewport. This allows you to select only the entities you want to edit.'

Reply:
Fixed.